controllers/v1/platform-roles.js

```diff
@@ -5,19 +5,18 @@
  * Description : platform Roles information. 
  */
 
- 
-  /**
-     * Forms
-     * @class
- */
+
+/**
+   * PlatformRolesExt
```

Comment:
@rakeshSgr - The filename above doesn't match and the description should start with caps

Reply:
resolved